grails-app/migrations/views/product-count-history.sql

```diff
@@ -1,19 +1,16 @@
 create or replace view product_count_history as
```

Comment:
I'm still confused why this view is needed if we have product_physical_count_history, which should already include the transactions from cycle counts. Can't we just pull from product_physical_count_history directly?

Reply:
Because when we do cycle counts by bin (0.9.5), I assume we probably won't have a product inventory (snapshot) transaction created for cycle counts (we'll just use adjustments). Using the snapshot transaction for these will be much more dangerous. 

At some point in the future, I would actually prefer to move in the opposite direction and use only cycle count records here (i.e. ignore the transactions altogether). If we didn't have to look at transaction data, then the candidates query would return in under a second.

Reply:
These are great questions, by the way. Thanks for putting in the effort to understand the view.